src/main/java/bruh/command/AddTaskCommand.java

```diff
@@ -0,0 +1,29 @@
+package bruh.command;
+
+import bruh.storage.Storage;
+import bruh.task.Task;
+import bruh.tasklist.TaskList;
+import bruh.ui.Ui;
+
+/**
+ * Represents a command which adds a task to the task list.
+ */
+public class AddTaskCommand extends Command {
+    private final Task taskToAdd;
+
+    /**
+     * Constructor for a command to add a task to the task list.
```

Comment:
According to the coding standards, I suppose the first sentence should start with something like "Instantiates" or "Constructs" instead of "Constructor...".

Reply:
Thanks for pointing this out! I took a look at the coding standards and this doesn't seem to be explicitly specified, so I went on to take a look at the [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment) (the coding standard recommends this for topics not covered) and it seems that it is okay for the summary fragment of the Javadoc to begin with a noun phrase.